prowler/providers/aws/services/organizations/organizations_service.py

```diff
@@ -0,0 +1,231 @@
+import json
+from typing import Optional
+
+from botocore.client import ClientError
+from pydantic import BaseModel
+
+from prowler.lib.logger import logger
+from prowler.lib.scan_filters.scan_filters import is_resource_filtered
+from prowler.providers.aws.aws_provider import generate_regional_clients
+
+
+################## Organizations
+class Organizations:
+    def __init__(self, audit_info):
+        self.service = "organizations"
+        self.session = audit_info.audit_session
+        self.audited_account = audit_info.audited_account
+        self.audit_resources = audit_info.audit_resources
+        global_client = generate_regional_clients(
+            self.service, audit_info, global_service=True
+        )
+        self.client = list(global_client.values())[0]
+        self.region = self.client.region
+        self.organizations = []
+        self.policies = []
+        self.delegated_administrators = []
+        self.__describe_organization__()
+
+    def __describe_organization__(self):
+        logger.info("Organizations - Describe Organization...")
+
+        try:
+            # Check if Organizations is in-use
+            try:
+                organization_desc = self.client.describe_organization()["Organization"]
+                organization_arn = organization_desc["Arn"]
+                organization_id = organization_desc["Id"]
+                organization_master_id = organization_desc["MasterAccountId"]
```

Comment:
For accessing a key from a dictionary is better to use `dict.get()` like `organization_desc.get("Arn")`. This way if the value is not present it not raises a `KeyError` and returns `None`. Also you can set a default value if `None` is raised using ``organization_desc.get("Arn", default_value)`.

Reply:
Please, change it for every dict access. Thanks!